internal/armada/server/submit.go

```diff
@@ -270,10 +271,15 @@ func (server *SubmitServer) SubmitJobs(ctx context.Context, req *api.JobSubmitRe
 
 	q, err := server.getQueueOrCreate(ctx, req.Queue)
 	if err != nil {
-		return nil, err
+		code := codes.Unknown
+		if e, ok := status.FromError(err); ok {
+			code = e.Code()
+		}
+
+		return nil, status.Errorf(code, "[SubmitJobs] couldn't get/make queue: %s", err)
```

Comment:
I think we have moved away from adding the function name into the error messages